core/state_processor.go

```diff
@@ -158,11 +157,6 @@ func ApplyTransactionWithEVM(msg *Message, config *params.ChainConfig, gp *GasPo
 	if msg.To == nil {
 		receipt.ContractAddress = crypto.CreateAddress(evm.TxContext.Origin, tx.Nonce())
 	}
-
```

Comment:
same, accessEvent is only for gas accounting, let's decouple the witness construction from the accessEvent now.